src/test/user.spec.js

```diff
@@ -133,104 +159,126 @@ describe('Test user login, signup and account verification', () => {
         });
     });
 
-    it.skip('it should login the user into an account', done => {
-      const user = {
-        email: 'ƒ@gmail.com',
-        password: 'password'
-      };
-      chai
-        .request(app)
+    it('should login the user into an account', async () => {
+      const user = await getUser();
+      user.email = 'sholabola@yahoo.com';
+      user.status = 'active';
+      user.password = 'password';
+
+      await createUser(user);
+
+      const res = await server()
         .post('/api/v1/auth/login')
-        .send(user)
-        .end((err, res) => {
-          expect(res.status).to.equal(200);
-          expect(res.body).to.have.property('status');
-          expect(res.body.data)
-            .to.have.property('status')
-            .to.equal('active');
-          done();
+        .send({
+          email: user.email,
+          password: user.password
         });
+
+      expect(res.status).to.equal(200);
+      expect(res.body).to.have.property('status');
+      expect(res.body.data)
+        .to.have.property('status')
+        .to.equal('active');
```

Comment:
show that a user is logged in by testing that a token was returned.

You can assert the validity of the token, the contents of the token etc. This is the login contract that the app exposes. This is how we know that a user is logged in.